platform-tests/pom.xml

```diff
@@ -1,277 +1,348 @@
-                            </classpathDependencyExclude>
-                        </classpathDependencyExcludes>
-                        <useSystemClassLoader>true</useSystemClassLoader>
-                        <useManifestOnlyJar>false</useManifestOnlyJar>
-                        <argLine>-Xmx${test.heap.size} -Dorg.bytedeco.javacpp.maxphysicalbytes=${test.offheap.size} -Dorg.bytedeco.javacpp.maxbytes=${test.offheap.size} </argLine>
-                        <forkedProcessTimeoutInSeconds>240</forkedProcessTimeoutInSeconds>
-                        <forkedProcessExitTimeoutInSeconds>240</forkedProcessExitTimeoutInSeconds>
-                        <parallelTestsTimeoutInSeconds>240</parallelTestsTimeoutInSeconds>
-                        <parallelTestsTimeoutForcedInSeconds>240</parallelTestsTimeoutForcedInSeconds>
-                        <forkCount>1</forkCount>
-                        <threadCount>1</threadCount>
-                        <perCoreThreadCount>false</perCoreThreadCount>
-                    </configuration>
-                </plugin>
-            </plugins>
-    </build>
-
-</project>
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.eclipse.deeplearning4j</groupId>
+    <artifactId>platofrm-tests</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <name>platform-tests</name>
+
+
+    <repositories>
+        <!-- Mainly for dokka/kotlinx. Not used outside of releases for kotlin related modules-->
+        <repository>
+            <id>jetbrains-kotlinx</id>
+            <url>https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven</url>
+            <name>Kotlinx repo</name>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+        </repository>
+
+
+        <repository>
+            <id>ossrh</id>
+            <name>Sonatype Nexus Snapshots</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>  <!-- Optional, update daily -->
+            </snapshots>
+        </repository>
+    </repositories>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
+        <dl4j.version>1.0.0-SNAPSHOT</dl4j.version>
+        <platform.classifier>windows-x86_64</platform.classifier>
```

Comment:
Did you forget to change this back, or is this intentionally tied to windows?

Reply:
@treo good point. Generally it's expected that this is specified on command line. The default is really anything that we need it to be. I cleaned up the dup maven.compiler/source target and changed this back to linux.